web/build.rs

```diff
@@ -16,7 +16,7 @@ fn main() {
     ] {
         println!("cargo:rerun-if-changed={tracked_file}");
     }
-    let vite_status = Command::new("pnpm")
+    let vite_status = Command::new("yarn")
```

Comment:
```suggestion
    let yarn_status = Command::new("yarn")
```

Also needs update a little further down.

Reply:
Yarn replaces pnpm, not vite.